### DISCUSSION TITLE
How can I remove the world name from the chat line [TownyChat]

### DISCUSSION THREAD

**Discussion by h4ppyboy** [TownyAdvanced/Towny#7002] [Q&A] [answered]
![image](https://github.com/TownyAdvanced/Towny/assets/75522441/44f80e7c-ca3b-4af8-9d59-7ad22d186aa3)
I don't want it to show world name, how to fix?

**Comment by LlmDl** [comment, marked as answer]
In order to edit your chat line you need to open your towny\settings\chatconfig.yml file. Once you have it open you can navigate to the channel_formats lines, find the global one. Then being to cross reference the tags used, with the tags in the chatconfig.yml's comments section.

Once you've customized the global channel_format line to your liking save the chatconfig.yml and use /townychat reload ingame.

**Comment by h4ppyboy** [comment]
ty friend!
